patches/src/rdkafka_conf.c.patch

```diff
@@ -1,18 +1,18 @@
 diff --git a/src/rdkafka_conf.c b/src/rdkafka_conf.c
-index e5c1415f..36b43572 100644
+index 9200af4c..314bb654 100644
 --- a/src/rdkafka_conf.c
 +++ b/src/rdkafka_conf.c
 @@ -33,6 +33,9 @@
  #include <stdlib.h>
  #include <ctype.h>
  #include <stddef.h>
-+#ifdef _ZOS
++#ifdef __MVS__
 +#include <strings.h>
```

Comment:
Can we remove strings.h here as well?

Reply:
I have removed strings.h includes from all the files